server/models/param_field.py

```diff
@@ -563,6 +573,7 @@ def from_dict(self, d: Dict[str, Any]) -> 'ParamField':
             placeholder=d.get('placeholder', ''),
             tab_parameter=d.get('tab_parameter', ''),
             visible_if=d.get('visible_if'),
+            child_parameters=d.get('parameters', []), # just `parameters` in YAML spec but child_parameters everywhere else
```

Comment:
You've told me you prefer `parameters` in the YAML spec to `child_parameters`. Is `parameters` better? If so, why? Otherwise, please rename to `child_parameters` here and in the schema.

Also, nit: nix the `, []` because we want `None` on non-List param fields.